<html><meta name="viewport" content="width=device-width, initial-scale=1.0"><head><h3>25/04/2019 02:40:02</h3></head><body><h2>cald</h2><br>  File "/usr/local/lib/python2.7/dist-packages/paho/mqtt/client.py", line 895, in reconnect
<br>    sock = socket.create_connection((self._host, self._port), source_address=(self._bind_address, 0))
<br>  File "/usr/lib/python2.7/socket.py", line 575, in create_connection
<br>    raise err
<br>socket.error: [Errno 111] Connection refused
<br><h2>cont</h2><br><h2>ener</h2><br>Traceback (most recent call last):
<br>  File "/home/pogo/script/Energy_pao.py", line 48, in <module>
<br>    if (crc8(alfa[2:8].decode("hex")) == int(alfa[10:12],16)):
<br>ValueError: invalid literal for int() with base 16: ''
<br></body></html>